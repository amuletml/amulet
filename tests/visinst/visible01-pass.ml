let foo : forall ('a : type) -> 'a -> 'a = fun x -> x

let bar = foo @{int} 1
and baz = foo ? 1
