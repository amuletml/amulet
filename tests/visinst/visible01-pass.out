foo : ('a : type) -> 'a -> 'a
bar : int
baz : int
